x/auth/client/rest/encode.go

```diff
@@ -42,6 +47,28 @@ func EncodeTxRequestHandlerFn(clientCtx client.Context) http.HandlerFunc {
 		txBytesBase64 := base64.StdEncoding.EncodeToString(txBytes)
 
 		response := EncodeResp{Tx: txBytesBase64}
+
+		// NOTE: amino is set intentionally here, don't migrate it
 		rest.PostProcessResponseBare(w, clientCtx, response)
 	}
 }
+
+func convertAndEncodeStdTx(txConfig client.TxConfig, stdTx types.StdTx) ([]byte, error) {
+	builder := txConfig.NewTxBuilder()
+
+	var theTx sdk.Tx
+
+	// check if we need a StdTx anyway, in that case don't copy
+	if _, ok := builder.GetTx().(types.StdTx); ok {
+		theTx = stdTx
+	} else {
```

Comment:
the else clause is not covered

Reply:
It is in proto unit tests.

Reply:
I think I can find a way to trigger it in both builds though.